src/assets/javascripts/components/content/code/mermaid/index.css

```diff
@@ -92,6 +92,11 @@ marker {
   stroke: var(--md-default-fg-color--lighter);
 }
 
```

Comment:
Looking good! Only thing missing is a comment on what this is exactly styling, see the other comments 😉

Reply:
Explanation: the CSS is very complex, so I try to add short and concise comments that a selector is targeting, so things are not accidentally removed or forgotten to be adapted.

Reply:
Of course, I thought the comment above would suffice, but we can be even more explicit :+1: